lib/foreman/renderer.rb

```diff
@@ -49,6 +50,16 @@ def host_param_false?(name)
       @host.params.has_key?(name) && Foreman::Cast.to_bool(@host.params[name]) == false
     end
 
+    def subnet_has_param?(subnet, param_name)
+      raise ::Foreman::Renderer::WrongSubnetError.new(N_("'%{object_name}' is a '%{object_class}', expected a subnet.") % { object_name: subnet.to_s, object_class: subnet.class.to_s}) unless subnet.is_a?(Subnet)
+      return !subnet.parameters.where(name: param_name).empty?
```

Comment:
I think it would be slightly more readable without negation and using `exists?` like this
`return `subnet.parameters.exists?(name: param_name)`

it also generates slightly more effective SQL query